man/fi_usnic.7.md

```diff
@@ -98,17 +98,30 @@ request a version rather than using the header provided
 `FI_EXT_USNIC_INFO_VERSION`. Although there is a version 1 of the extension,
 its use is discouraged, and it may not be available in future releases.
 
+## Fabric Extension: getinfo: compatibility issues
```

Comment:
Minor nit: Make this a `###` (i.e., a sub of the larger `Fabric Extension: getinfo` section).  And just call it "Compatibility issues".